telebot/util.py

```diff
@@ -56,8 +57,8 @@ def run(self):
                     self.done_event.set()
                 except Queue.Empty:
                     pass
-                except:
-                    logger.debug("Exception occurred")
+                except Exception as e:
+                    logger.error(type(e).__name__ + " occurred, args=" + str(e.args) + "\n" + traceback.format_exc())
```

Comment:
This could be replaced with `logging.exception("Error occured")`, which itself will error-log the exception type, message and its traceback. Just for the sake of code simplicity.